snap/snapcraft.yaml

```diff
@@ -0,0 +1,32 @@
+name: lp-to-jira
+base: core18
+version: '0.1'
+summary: Command Line Interface to import Launchpad content into JIRA
+description: |
+  lp-to-jira allows you to import bug from a launchpad project into a JIRA
```

Comment:
If we ever add support for operations other than just bugs (something along the lines of syncing LP Merge Proposals), this description won't necessarily be accurate.

But I talked with my team and there wasn't really interest in adding functionality for MPs, so this is just a "maybe someday" thought.

Reply:
Thanks @genet022 everything is looking good except I'm having an issue with the launchpadlib which doesn't seem to like being run inside a snap. it keeps on asking for a new token. I'm investigating...